CONTRIBUTING.md

```diff
@@ -11,7 +11,36 @@ No software is bug free. So, if you got an issue, follow these steps:
   * Clear title (make is shorter if possible).
   * Describe the issue in clear language.
   * Share error logs, screenshots and etc.
-  * To speed up the issue fixing process, send us a sample repo with the issue you faced.
+  * To speed up the issue fixing process, send us a sample repo with the issue you faced:
+  
+### Reproductions
+
+The best way to help figure out an issue you are having is to produce a minimal reproduction. A good way to do that is using Create React App:
+
+```bash
+# if necessary
+npm install -g create-react-app getstorybook
+
+create-react-app reproduction
+cd reproduction
+getstorybook
+
+# make changes to try and reproduce the problem, such as adding components + stories
+yarn start storybook
+
+# see if you can see the problem, if so, commit it:
+git init
+git add -A
+git commit -m "reproduction for issue #123"
+
+# create an new repository on github to host the reproduction, then:
+git remote add origin https://github.com/<your-username>/<repo-name>
+git push -u origin master
+```
+
+If you follow that process, you can then link to the github repository in the issue. See https://github.com/storybooks/react-storybook/issues/708#issuecomment-290589886 for an example.
```

Comment:
There is an example in the linked comment. We can change this later when somebody submits an issue with an attached repro perhaps?